src/test/fuzz/tx_pool.cpp

```diff
@@ -286,7 +286,7 @@ FUZZ_TARGET(tx_pool_standard, .init = initialize_tx_pool)
         std::set<CTransactionRef> removed;
         std::set<CTransactionRef> added;
         auto txr = std::make_shared<TransactionsDelta>(removed, added);
-        RegisterSharedValidationInterface(txr);
+        g_setup->m_node.validation_signals->RegisterSharedValidationInterface(txr);
```

Comment:
nit:
```suggestion
        node.validation_signals->RegisterSharedValidationInterface(txr);
```

Same for the others below.